app/views/reviews/_index.html.erb

```diff
@@ -0,0 +1,25 @@
+<% @reviews.each do |review| %>
+  <p><%= review.rating %>/5 --
+  <%= review.body %>
+  Votes: <%= review.votes.tally %>
+  </p>
+
+  <% if review.votes.find_by(user: current_user) && vote_value = 1 %>
+    <%= link_to "Upvote", review_downvote_path(review.id, review.user_vote(current_user)), method: :patch%>
```

Comment:
Should this be review_upvote_path instead? Also why are you checking for vote_value = 1 for both upvotes and downvotes when downvotes are valued at -1?